clientgui/DlgItemProperties.cpp

```diff
@@ -85,6 +96,10 @@ CDlgItemProperties::CDlgItemProperties(wxWindow* parent) :
         break;
     }
 
+#ifdef __WXMAC__
+     m_txtInformation->SetEditable(true);
```

Comment:
I'm not sure about this code. I have no possibility to test this on Mac.
Actually I got this piece of code from @CharlieFenton.
@CharlieFenton could you please comment whether this line is necessary or not? 
Thanks